misk/src/main/kotlin/misk/web/dashboard/WebTabResourceModule.kt

```diff
@@ -26,24 +27,44 @@ import misk.web.resources.StaticResourceEntry
  * @property resourcePath JVM path for non-Development environment static resources (includes `classpath:/` prefix)
  */
 class WebTabResourceModule(
-  val environment: Environment,
+  private val isDevelopment: Boolean,
```

Comment:
Pass in the Deployment instead?

Reply:
Keeping for backwards compatibility until `Environment` is deleted. After that, will replace this and have a single ctor (the one currently on line 37).